catalog-info.yaml

```diff
@@ -177,6 +177,39 @@ spec:
         search-extract-and-transform: {}
         search-productivity-team: {}
 
+# Dockerfiles nightly images build pipeline
+---
+apiVersion: "backstage.io/v1alpha1"
+kind: "Resource"
+metadata:
+  name: "connectors-dockerfiles-nightly"
+  description: "Nightly Connectors Dockerfiles Build"
+spec:
+  type: "buildkite-pipeline"
+  owner: "group:search-extract-and-transform"
+  system: "buildkite"
+  implementation:
+    apiVersion: "buildkite.elastic.dev/v1"
+    kind: "Pipeline"
+    metadata:
+      name: "connectors-dockerfiles-nightly"
+      description: "Nightly Connectors Dockerfiles Build"
+    spec:
+      pipeline_file: ".buildkite/dockerfiles-pipeline.yml"
+      provider_settings:
+        trigger_mode: "none"
+      repository: "elastic/connectors"
+      schedules:
+        Daily main:
+          branch: main
+          cronline: '@daily'
```

Comment:
wondering if we should consider running this less frequently? I think it would depend on what @elastic/search-extract-and-transform prefer, but the scenario I picture is something like:
* this job runs once a week, or maybe even once every two weeks
* if Trivy identifies any vulnerabilities, a message is sent to a Slack channel, or a GitHub issue is created

let's see what folks think 👍

Reply:
> * if Trivy identifies any vulnerabilities, a message is sent to a Slack channel, or a GitHub issue is created

I think these notifications could potentially be a duplicate of existing Snyk issues detected on the official container image. What's the value added by these new notifications?